### ISSUE TITLE
Google Maps v2 crashes on targetSDK 28

### ISSUE THREAD

**Issue by moving-bits** [cgeo/cgeo#7841]
When changing targetDSK to 28 manually on today's master branch, c:geo directly crashes when starting live map with Google Maps enabled.

Logcat shows:

```
2019-09-13 18:02:19.681 5282-5370/cgeo.geocaching E/AndroidRuntime: FATAL EXCEPTION: Thread-4
    Process: cgeo.geocaching, PID: 5282
    java.lang.NoClassDefFoundError: Failed resolution of: Lorg/apache/http/ProtocolVersion;
        at bz.b(:com.google.android.gms.dynamite_mapsdynamite@19056100@19.0.56 (120700-262933554):3)
        at ca.a(:com.google.android.gms.dynamite_mapsdynamite@19056100@19.0.56 (120700-262933554):3)
        at cc.a(:com.google.android.gms.dynamite_mapsdynamite@19056100@19.0.56 (120700-262933554):14)
        at com.google.maps.api.android.lib6.drd.ak.a(:com.google.android.gms.dynamite_mapsdynamite@19056100@19.0.56 (120700-262933554):4)
        at be.a(:com.google.android.gms.dynamite_mapsdynamite@19056100@19.0.56 (120700-262933554):22)
        at be.run(:com.google.android.gms.dynamite_mapsdynamite@19056100@19.0.56 (120700-262933554):8)
     Caused by: java.lang.ClassNotFoundException: Didn't find class "org.apache.http.ProtocolVersion" on path: DexPathList[[zip file "/data/user_de/0/com.google.android.gms/app_chimera/m/0000000e/MapsDynamite.apk"],nativeLibraryDirectories=[/data/user_de/0/com.google.android.gms/app_chimera/m/0000000e/MapsDynamite.apk!/lib/x86, /system/lib, /system/product/lib]]
        at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:196)
        at dalvik.system.DelegateLastClassLoader.loadClass(DelegateLastClassLoader.java:137)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:312)
        at bz.b(:com.google.android.gms.dynamite_mapsdynamite@19056100@19.0.56 (120700-262933554):3) 
        at ca.a(:com.google.android.gms.dynamite_mapsdynamite@19056100@19.0.56 (120700-262933554):3) 
        at cc.a(:com.google.android.gms.dynamite_mapsdynamite@19056100@19.0.56 (120700-262933554):14) 
        at com.google.maps.api.android.lib6.drd.ak.a(:com.google.android.gms.dynamite_mapsdynamite@19056100@19.0.56 (120700-262933554):4) 
        at be.a(:com.google.android.gms.dynamite_mapsdynamite@19056100@19.0.56 (120700-262933554):22) 
        at be.run(:com.google.android.gms.dynamite_mapsdynamite@19056100@19.0.56 (120700-262933554):8) 
2019-09-13 18:02:20.102 1897-2006/? E/InputDispatcher: channel '7d0883 cgeo.geocaching/cgeo.geocaching.maps.google.v2.GoogleMapActivity (server)' ~ Channel is unrecoverably broken and will be disposed!
2019-09-13 18:02:20.169 1897-2006/? E/InputDispatcher: channel '3177ec8 cgeo.geocaching/cgeo.geocaching.MainActivity (server)' ~ Channel is unrecoverably broken and will be disposed!
```

This is on Android 10, started in an AS emulator session.

**Comment by moving-bits** [comment]
According to https://stackoverflow.com/questions/50461881/java-lang-noclassdeffounderrorfailed-resolution-of-lorg-apache-http-protocolve solutions are to
- either update the play-services-maps library to at least version 16.1 (eg: `com.google.android.gms:play-services-maps:16.1.0`)
- or to add `<uses-library android:name="org.apache.http.legacy" android:required="false" />` to `AndroidManifest.xml`

**Comment by moving-bits** [comment]
A fix for this will be part of the switch to targetSDK28 PR #7653.

**Comment by moving-bits** [comment]
fixed with 4643293
